suite-common/wallet-utils/src/exportTransactions.ts

```diff
@@ -128,62 +136,112 @@ const prepareContent = (data: Data): Fields[] => {
     return transactions
         .map(formatAmounts(coin))
         .flatMap(t => {
+            if (getIsZeroValuePhishing(t)) {
```

Comment:
is there a change the condition is incorrect and it's not phishing? Should that transaction still be included? @sime

Reply:
I would leave it here for now. However, I think that we should remake `getIsZeroValuePhishing` condition. #7907